tools/install_python_deps.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+
+import subprocess
+import os
+from os.path import join as pjoin
+
+with open(pjoin(os.path.dirname(__file__), '../requirements.txt')) as f:
+    deps = f.readlines()
+
+for dep in deps:
```

Comment:
Combine as `for dep in open(pjoin(os.path.dirname(__file__), '../requirements.txt')):`?